ecosystem/gstreamer_plugin/gst_mtl_st20p_rx.c

```diff
@@ -511,3 +563,45 @@ GST_PLUGIN_DEFINE(GST_VERSION_MAJOR, GST_VERSION_MINOR, mtl_st20p_rx,
                   "software-based solution designed for high-throughput transmission",
                   plugin_init, PACKAGE_VERSION, GST_LICENSE, GST_PACKAGE_NAME,
                   GST_PACKAGE_ORIGIN)
+
+static int gst_mtl_st20p_rx_query_ext_frame(void* priv, struct st_ext_frame* ext_frame,
+                                            struct st20_rx_frame_meta* meta) {
+  Gst_Mtl_St20p_Rx* s = (Gst_Mtl_St20p_Rx*)priv;
```

Comment:
why not just pass it as Gst_Mtl_st20p_Rx from the get go  ?

Reply:
This is callback function that have to follow MTL naming convention 
`  int (*query_ext_frame)(void* priv, struct st_ext_frame* ext_frame, struct st20_rx_frame_meta* meta); `